src/app/integrations/qbo/qbo-main/qbo-dashboard/qbo-dashboard.component.ts

```
@@ -72,6 +72,8 @@
 
   importCodeFields: any;
 
+  private destroy$ = new Subject<void>(); // why this subject? -> to stop polling when the component is destroyed/page closed.
```

ESLint

[error] Comments should not begin with a lowercase character.

(capitalized-comments)